### PR TITLE
[VL][DOC] Add ABFS doc

### PR DESCRIPTION
**Pull request by acvictor**
Add documentation for ABFS with Gluten using Velox backend.

### CODE REVIEW

**Comment by acvictor** [comment]
@PHILO-HE can you please review? Thank you!

**Comment by zhli1142015** [comment]
cc @gaoyangxiaozhu , thanks

**Review by PHILO-HE — COMMENTED** [review]
Just two trivial comments. Thanks!

File: docs/get-started/Velox.md [hunk 1]
```diff
@@ -191,6 +191,19 @@ Here are two steps to enable kerberos.
 
 The ticket cache file can be found by `klist`.
 
+## Azure Blob File System (ABFS) support
+
+Velox supports ABFS with the open source [Azure SDK for C++](https://github.com/Azure/azure-sdk-for-cpp) and Gluten uses the Velox ABFS connector to connect with ABFS.
+The build option for ABFS (enable_abfs) must be set to enable this feature as listed below
```
[the remark this answers is not in the document]
Comment (acvictor) [reply docs/get-started/Velox.md]: Added, thanks!

**Comment by acvictor** [comment]
> Just two trivial comments. Thanks!

Thank you @PHILO-HE!

**Comment by PHILO-HE** [comment]
Let's wait for the review feedback from @gaoyangxiaozhu.

**Pull request merged**